src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -145,7 +148,7 @@ public void StartClimberRPM(){
       //}
       //else
       //{
-        m_climber.set(0.7);
+        m_climber.set(0.1);
```

Comment:
Why are you changing this from 0.7 to 0.1?

Reply:
We like to start at a slow speed and will increase the speed during the test for safety.